dynamiq/components/retrievers/pinecone.py

```diff
@@ -42,13 +42,15 @@ def run(
         self,
         query_embedding: list[float],
         exclude_document_embeddings: bool = True,
+        content_key: str | None = None,
```

Comment:
Please add a new param at the end to avoid breaking changes as users could pass params without key.